### PR TITLE
Api human readable day

### PR DESCRIPTION
**Pull request by okwme**
Added human readable version of date to all time record using `SpeedScore`. Couldn't figure out how to add to the all time fastest days, @psugihara do you know how?

### CODE REVIEW

**Comment by psugihara** [comment]
i'll take a swing

**Comment by psugihara** [comment]
kk think I got it, claude is a genius

File: server/src/leaderboard.ts [hunk 1]
```diff
@@ -185,6 +185,7 @@ WITH current_streaks AS (
         ) AS subquery
         GROUP BY
             player, streak
+        HAVING MAX(day) = ${today} -- ensure the streak is current
```
Comment (psugihara) [inline server/src/leaderboard.ts:188]: @okwme I *think* this makes sure the streak ends today

**Comment by okwme** [comment]
I added another line to make sure people who are part of the current streak but haven't yet submitted _today_ are included. If they miss the current day, the next day they will not be included until they submit (at which point they will have a streak of 1)

also baddie cores shouldn't loop across edge.  
actually i saw you posted another question in another issue [here](https://github.com/trifle-labs/anybody-problem/pull/189#issuecomment-2223396605) was this with regard to the visual of the hero body looping or something else? I didn't see anything awry in the image otherwise. Wasn't sure if you hadn't seen visual looping before but thought we had it enabled when you would have been working on it. Or I'm missing something else in the image...

**Comment by psugihara** [comment]
I think I just thought it looked kind of funky. The core/face looks missing on the left one then it flashes in.

**Pull request merged**